k8s-bench/tasks/resize-pvc/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+kubectl delete namespace resize-pv --ignore-not-found
+kubectl create namespace resize-pv
+
+kubectl apply -f ./storage-pvc.yaml
+kubectl apply -f ./storage-pod.yaml
```

Comment:
can we pl. move these static yamls under `artifacts` directory. We did that for some evals, we will standardize on that.